### DISCUSSION TITLE
Add ability to retrieve a Custom Field via the Org Browser

### DISCUSSION THREAD

**Discussion by mpaquinfionta** [forcedotcom/salesforcedx-vscode#5791] [Ideas]
As of right now the org browser only support retrieving the full object. This includes, the object-meta.xml file, all the fields, listViews and recordTypes. This is problematic for three reasons.

1. It really limits our ability to be flexible with what is under version control. For example: we want to track custom fields but not page layout.
2. It makes it hard to deploy specific fields to an org without overwriting everything for that object.
3. Having to manually "clean" the object folder every time we retrieve the metadata is quite cumbersome.

It would be great if we could target specific fields(and other file type) and just retrieve them. 

As of right now the only two way this is possible is via the CLI using the following command: sfdx force:source:retrieve -m CustomField:Account.Name or by using a third party plugin like this: https://marketplace.visualstudio.com/items?itemName=VignaeshRamA.sfdx-package-xml-generator . The first solution is a non starter for non-developers or people that are not familiar with the CLI and the second solution works great but I would love that feature to be part of the already existing Org Browser.

Thank you,

Max Paquin

**Comment by dannybusiness** [comment]
This would be really useful and time saving functionality.

**Comment by itaishmida** [comment]
Love this idea
Any idea if this is being worked on? is on the road map?

**Comment by mshanemc** [comment]
closed and merged in favor of https://github.com/forcedotcom/salesforcedx-vscode/discussions/3938
